src/module.ts

```diff
@@ -175,12 +181,19 @@ class Ctrl extends PanelCtrl {
   private async _getGrafanaAPIInfo() {
     this._user = await this._getCurrentUser();
     for(let panel of this.panels) {
-      let datasourceName = panel.datasource;
-      if(!datasourceName) {
+      if (panel.type === 'corpglory-data-exporter-panel') {
         continue;
       }
-      let datasource = await this._getDatasourceByName(datasourceName);
-      this._datasourceTypes[panel.id] = datasource.type;
+
+      if (!panel.datasource) {
```

Comment:
```suggestion
      if(!panel.datasource) {
```

we don't add space before brackets in our codestyle